File: Java/Ejercicio/src/Ejercicio5.java
package Ejercicio.src;

import java.util.Scanner;

public class Ejercicio5 {
    public static void main(String[] args) throws Exception {
        Scanner keyboard = new Scanner(System.in);
        int n = 0;
        System.out.println("please enter the value in A");
        n = keyboard.nextInt();
        int number = n % 2;
        if (number == 0) {
            System.out.println("the number " + n + " is pair");
        } else {
            System.out.println("the number " + n + " is odd");
        }
        keyboard.close();
    }
}

File: Javascript/Ejercicio/Ejercicio5.js
    
      process.stdout.write("please enter the value in A ");
      process.stdin.on('data',async function(data1) {
        const n = await parseInt(data1.toString());
    
        let number = n % 2;
        
        if (number == 0) {
            console.log(`the number ${n} is pair`);
        } else {
            console.log(`the number ${n} is odd`);
        }
        process.exit();
    });
    
